sapi/phpdbg/phpdbg_opcode.h

```diff
@@ -21,7 +21,6 @@
 
 #include "zend_types.h"
 
-char *phpdbg_decode_opline(zend_op_array *ops, zend_op *op);
```

Comment:
Looks like print_opline and print_opline_ex are the same (even before this change).

Reply:
Fixed in 6a3898195f29dd11b98f3a3f46d10c5ec732f881